libyaml_q_declaration.md

```diff
@@ -137,10 +141,16 @@ This library does not have external dependencies.
 
 ## Platform Support [6]
 
-This library does not state support for any specific platform.
+This library does not state support for any specific platform. But this is build in the ROS 2 buildfarm for atll Tier 1 platforms:
+
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/libyaml_vendor/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/libyaml_vendor/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/libyaml_vendor/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/libyaml_vendor/)
 
 ## Security [7]
 
 ### Vulnerability Disclosure Policy [7.i]
 
-This package does not yet have a Vulnerability Disclosure Policy.
+This package does not yet have a Vulnerability Disclosure Policy. But it's defined in the vendored Quality Declaration.
```

Comment:
These 2 sentences seem to be contradicting each other. How about something along the lines of "The libyaml library does not yet have a Vulnerability Disclosure Policy, but for issues affecting ROS, see the policy defined in the vendor package."